app/src/androidTest/java/com/github/fribourgsdp/radio/mockimplementations/MockLobbyActivities.kt

```diff
@@ -27,6 +27,10 @@ class WorkingLobbyActivity : LobbyActivity() {
 
         startActivity(intent)
     }
+
+    override fun loadUser(): User {
```

Comment:
I think it would be a good idea to also set the id explicitly, as it may be used in the activity